x-pack/legacy/plugins/maps/public/connected_components/layer_panel/style_settings/_style_settings.scss

```diff
@@ -0,0 +1,3 @@
+.mapStyleSettings__fixedBox {
+    width: 120px;
```

Comment:
Even though this is a large number, still consider making it a multiple of our sizing variables.

Reply:
Thanks! I made it a multiple of our sizing variables.